data/blog/electron-13-0.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Electron 13.0.0
+author:
+- VerteDinde
+- clavin
+- georgexu99
+date: '2021-05-25'
+---
+
+Electron 13.0.0 has been released! It includes upgrades to Chromium `91`, V8 `x.x` and Node.js `14.x`. We've added {highlighted changes}, and general improvements. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 13.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). Continue reading for details about this release, and please share any feedback you have!
+
+## Notable Changes
+
+### Stack Changes
+
+* Chromium `91`
+    * [New in Chrome 88](https://developer.chrome.com/blog/new-in-chrome-88/)
```

Comment:
```suggestion
    * [New in Chrome 91](https://developer.chrome.com/blog/new-in-chrome-91/)
```

Reply:
The new in chrome 91 link is not live yet, but pretty sure they're following this URL convention